mousestyles/path_index.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function, absolute_import, division
+
+import numpy as np
+
+
+def path_index(movement, stop_threshold, min_path_length):
+    r"""
+    Return a list object containing start and end indices
+    for a specific movement. Each element in the list is
+    a list containing two indices: the first element is
+    the start index and the second element is the end index.
+
+    Parameters
+    ----------
+    movement : pandas.DataFrame
+        CT, CX, CY coordinates and homebase status
+        for the unique combination of strain, mouse and day
+    stop_threshold : int
+        positive integer indicating the path cutoff criteria
+        if the time difference between two observations is
+        less than this threhold, they will be in the same path
+
+    min_path_length : int
+        positive integer indicating how many observations in
+        a path
+
+    Returns
+    -------
+    paths index : a list containing the indices for all paths
+
+    Examples
+    --------
+    >>> movement = data.load_movement(1, 2, 1)
+    >>> paths = path_index(movement, 1, 1)
+    [[0, 2], [6, 8], [107, 113], [129, 131], [144, 152]]
+    """
+    # check if all inputs are positive integers
+    conditions_value = [stop_threshold <= 0, min_path_length <= 0]

```

Comment:
0 is an unacceptable input. When one of the values is equal to 0, the `conditions_value` will have a True in it.  It will raise the value error (see line 41 & 42).